rhombus/scribblings/ref-syntax-class.scrbl

```diff
@@ -33,7 +37,12 @@
  @rhombus[::]. The @rhombus[pattern] subform is optional in the sense
  that pattern alternatives can be inlined directly in the
  @rhombus[syntax.class] form (but the @rhombus[pattern] subform makes
- room for additional subforms in the future).
+ room for additional subforms in the future). The optional
+ @rhombus[description] subform can be used before @rhombus[pattern] to define
+ the description of the syntax class. This description is used to produce 
+ clearer error messages when a term is rejected by the syntax class. 
+ It takes a block with an expression that may evaluate to a string or 
+ @rhombus[#false].
```

Comment:
Maybe start a new paragraph for `The optional @rhombus[description]`? I'd use `must` instead of `may`. With the other changes, I'd write `a block whose result is` (since there's not just one expression). Clarify that `#false` is equivalent to not specifying `description`, which means that the error message is not customized.